CONTRIBUTING.md

```diff
@@ -92,19 +94,19 @@ This chart describes these simple transformations:
 +----------------------+            +---------------------------+
 ```
 
-##### Example `New-MarkdownHelp`
+### Example `New-MarkdownHelp`
 
-A user creates a platyPS Markdown for the first time with `New-MarkdownHelp`: 
+A user creates a platyPS Markdown for the first time with `New-MarkdownHelp`:
 
-```
+```powershell
 New-MarkdownHelp -Command New-MyCommandHelp
 ```
 
 Under the hood, the following tranformations happen:
 
 [MAML XML file] --> [Help Object + Get-Command object] --> [MAML Model] --> [Markdown file]
 
-# Making a new release
+## Making a new release
```

Comment:
Why changing header level? I think release section is outside of `Contributing to platyPS`

Reply:
Shouldn't there be only one level one header at the top of the document? Releasing is still contributing, but will change it if you like.

Reply:
No, it's a platyPS specific things about one header, markdown in general doesn't care